src/test/pythonnode_test.cpp

```diff
@@ -1187,6 +1187,24 @@ TEST_F(PythonFlowTest, PythonCalculatorTestSingleInTwoOutTwoParallelExecutorsWit
 }
 
 TEST_F(PythonFlowTest, ConverterWithMissingTagMap) {
+    ConstructorEnabledModelManager manager{"", getPythonBackend()};
+    std::string testPbtxt = R"(
+    input_stream: "OVTENSOR:in"
+    output_stream: "OVMS_PY_TENSOR:out"
+        node {
+            name: "pythonNode1"
+            calculator: "PyTensorOvTensorConverterCalculator"
+            input_stream: "OVTENSOR:in"
```

Comment:
TODO: Add test with:
```
[type.googleapis.com / mediapipe.PyTensorOvTensorConverterCalculatorOptions]: {

                }
```

Reply:
Could you also add such test for PythonExecutorCalculator?

Reply:
If this is specific mediapipe behavior then IMO it doens't make sense to multiply such tests. Maybe add single test to mediapipe_framework that will show that such formating of options is not allowed?